napari/layers/vectors/_tests/test_vectors.py

```diff
@@ -243,7 +243,7 @@ def test_adding_properties():
         layer.properties = properties_2
 
     # adding properties with the wrong length should raise an exception
-    bad_properties = {'vector_type': np.array(['A'])}
+    bad_properties = {'vector_type': np.array(['A', 'B'])}
```

Comment:
Note that without adding an extra element, this did not fail because pandas will successfully broadcast a column of length 1 to match any length column (i.e. 1 -> 10 works, but 2 -> 10 doesn't). I think that's fine, but just noting for reference.